Framework/Core/include/Framework/ASoAHelpers.h

```diff
@@ -1348,7 +1348,7 @@ auto combinations(const BP& binningPolicy, int categoryNeighbours, const T1& out
   }
 }
 
-template <typename... T2s>
```

Comment:
This concept can probably be beneficial independently and it should be trivial to implement. This would allow us to remove a few lines from this diff.

Reply:
I'm not sure I understand your point. You mean leaving a compatibility concept alias `is_soa_filtered_v` and not adding the restrictions in the templates for now, so the file can be left unchanged?

Reply:
No, I mean the opposite. If we have a concept which makes sense already now, I would introduce it and use it, so that we get it out of the way.

Reply:
Implemented in #13679